tests/core/util/test_keychain.py

```diff
@@ -165,6 +166,28 @@ def test_bip39_test_vectors(self):
             assert bytes_to_mnemonic(entropy_bytes) == mnemonic
             assert mnemonic_to_seed(mnemonic) == seed
 
+    def test_bip39_test_vectors_short(self):
+        """
+        Tests that the first 4 letters of each mnemonic phrase matches as if it were the full phrase
+        """
+        with open("tests/util/bip39_test_vectors_short.json") as f:
```

Comment:
This should be loading a resource probably, or at least working relative to some known path like `tests.__file__`.  This would be the minimal.  `pkg_resources` or `importlib.metadata` would be the bigger thing, but...  I need to do some work around those.

```suggestion
        with pathlib.Path(tests.__file__).joinpath("util", "bip39_test_vectors_short.json") as f:
```